static/app/components/list/utils.tsx

```diff
@@ -40,7 +40,9 @@ const numericStyle = (
             height: 24px;
             font-weight: ${theme.fontWeightNormal};
             font-size: ${theme.fontSizeSmall};
-            background-color: ${theme.yellow300};
+            background-color: ${theme.isChonk
+              ? (theme as DO_NOT_USE_ChonkTheme).colors.chonk.yellow400
```

Comment:
let’s do the `isChonkTheme` assert function to avoid the type assertion we need to do in a couple of places.

Reply:
Yeah, let me followup with that and fix it in a bunch of other places too